src/components/ExpenseForm.js

```diff
@@ -30,26 +38,41 @@ export default function ExpenseForm({ addToExpenses }) {
           name="title"
           required={true}
           autoComplete="off"
-          maxlength="30"
+          autoFocus
+          maxLength="30"
           placeholder="What did you spend money for?"
           value={inputValue.title}
-          ref={inputRef}
           onChange={event =>
             setInputValue({ ...inputValue, title: event.target.value })
           }
         />
       </label>
+      <label>
```

Comment:
One of the coaches remarked this in my code: apparently it's better for accessibility if you don't nest your input in your label, but rather use `<label htmlFor="description"/> <input id="description" .... >` and so on.

Reply:
Good to know, will edit it accordingly!